pulsar-sql/presto-pulsar/pom.xml

```diff
@@ -107,12 +107,6 @@
             <version>${presto.version}</version>
         </dependency>
 
-        <dependency>
```

Comment:
why is the pulsar client removed here?  The pulsar presto connector need the admin client.

Reply:
```
<dependency>
    <groupId>${project.groupId}</groupId>
    <artifactId>pulsar-client-admin-original</artifactId>
    <version>${project.version}</version>
</dependency>
```
This dependency already exists, it contains `pulsar-client-original`.